blurring_as_a_service/pre_inference_pipeline/source/workload_splitter.py

```diff
@@ -40,10 +46,7 @@ def create_batches(data_folder, number_of_batches, output_folder, execution_time
         execution_time: str
             Datetime containing when the job was executed. Used to prefix the files name.
         """
-        # Get all image paths within the input folder
         image_paths = get_image_paths(data_folder)
-
-        # Ensure number_of_batches is not greater than the number of images
         if number_of_batches > len(image_paths):
             logger.warning(
                 "Number of batches is greater than the number of images. Setting number_of_batches to 1."
```

Comment:
Would it make more sense to set the number of batches to the number of images? For example, there are 999 images and the user wanted 1000 batches. What is closer to the request: return 999 batches of 1 image, or return 1 batch of 999 images?